data/strings/strings.txt

```diff
@@ -15307,7 +15307,47 @@
     tags = android,ios
     comment = The second part of the editor_edit_place_name_hint to explain that name should be entered in a local language, see https://wiki.openstreetmap.org/wiki/Key:name
     en = As it is written in the local language
```

Comment:
Maybe just "In the local language"?

```
    en = In the local language
    af = In plaaslike taal
    ar = باللغة المحلية
    az = Yerli dildə
    be = На мясцовай мове
    bg = На местния език
    ca = En llengua local
    cs = V místním jazyce
    da = På det lokale sprog
    de = In der Landessprache
    el = Στην τοπική γλώσσα
    es = En la lengua local
    et = Kohalikus keeles
    eu = Bertako hizkuntzan
    fa = به زبان محلی
    fi = Paikallisella kielellä
    fr = Dans la langue locale
    he = בשפה המקומית
    hi = स्थानीय भाषा में
    hu = A helyi nyelven
    id = Dalam bahasa setempat
    it = Nella lingua locale
    ja = 現地語で
    ko = 현지 언어로
    lt = Vietos kalba
    mr = स्थानिक भाषेत
    nb = På det lokale språket
    nl = In de plaatselijke taal
    pl = W lokalnym języku
    pt = Na língua local
    pt-BR = No idioma local
    ro = În limba locală
    ru = На местном языке
    sk = V miestnom jazyku
    sv = På det lokala språket
    sw = Kwa lugha ya kienyeji
    th = ในภาษาถิ่น
    tr = Yerel dilde
    uk = Місцевою мовою
    vi = Bằng ngôn ngữ địa phương
    zh-Hans = 当地语言
    zh-Hant = 用當地語言
```

Reply:
this is also a valid option in "es".

Reply:
I still prefer "As it is written...", because it gives people the hint that they should just write down what's on the sign of the shop. But I don't have a strong opinion on this, so we can change it if you prefer "In the local language"